assets/js/payment-methods-demo/express-payment/apple-pay/apple-pay-express.js

```diff
@@ -34,8 +34,8 @@ import {
 import { __ } from '@wordpress/i18n';
 
 /**
- * @typedef {import('./type-defs').Stripe} Stripe
- * @typedef {import('./type-defs').StripePaymentRequest} StripePaymentRequest
+ * @typedef {import('../../stripe-utils/type-defs').Stripe} Stripe
```

Comment:
Will these typedefs be shared? `StripePaymentRequest` in Apple Pay.

Reply:
I might not be addressing your question but if you're asking if the type-defs will be shared across payment methods, Apple Pay is a part of the stripe gateway plugin, so the type-defs will just be a part of that plugin (whether that team actually uses the type-defs or not is up to them)

Reply:
ah sorry my bad :)